hpi-swt2/lecture-portal
# Create pull_request_template.md

This PR  adds a PR template.

Testing steps:
- unknown
- maybe merge this branch into one of your feature branches and try to create a pr. The pr should initially have the template.

Acceptance Criteria:
- The template should work and should contain al relevant information.
